src/components/Event/Event.visual.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { VisualTestContainer } from '../../../test/visual/VisualTestContainer';
+import { Event } from './';
+
+class EventVisual extends React.Component<any> {
+  render() {
+    return (
+      <VisualTestContainer>
+        <Event {...this.props} />
+      </VisualTestContainer>
+    );
+  }
+}
+
+const tests = [
+  {
+    describe: 'basic',
+    its: [
+      {
+        it: 'default',
+        props: {},
+      },
+    ],
+  },
+];
```

Comment:
Let's add more use cases to take screenshots of 
and preferably use `storybook-snapper` like you did with the cell component